pkg/convert/xliff12.go

```diff
@@ -50,7 +50,7 @@ type context struct {
 }
 
 // FromXliff12 converts serialized data from the XML data in the XLIFF 1.2 format into a model.Messages struct.
-func FromXliff12(data []byte) (model.Messages, error) {
+func FromXliff12(data []byte, original bool) (model.Messages, error) {
```

Comment:
I'm not sure how to use original param in xlliff formats because it is already in use, so should we ignore it?

Reply:
I think there is sufficient info to determine the originality of the file, e.g. there is no target language in file. So original parameter could be ignored. 
```suggestion
func FromXliff12(data []byte, _ bool) (model.Messages, error) {
```

Reply:
Let's keep `original` name for clarity and intent. It might be required to process xliff in the future.

Reply:
@DarjaLeonova can you please add a comment to all `From...` where `original` is ignored?

Reply:
sure, I will add now